### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by geeknoid**

### CODE REVIEW

**Comment by howardjohn** [comment]
lgtm seems codeowners still aren't applying here though for some reason

File: CODEOWNERS [hunk 1]
```diff
@@ -1 +1,6 @@
-*   @geeknoid, @rshriram, @andraxylia, @costinm, @diemtvu, @frankbu, @GregHanson, @mandarjog, @ozevren, @wattli
+*               @istio/technical-oversight-committee
+Makefile*       @istio/wg-test-and-release-maintainers
+*.md            @istio/wg-test-and-release-maintainers
+common/         @istio/wg-test-and-release-maintainers
```
Comment (howardjohn) [inline CODEOWNERS]: should be `/common/`, etc. Otherwise `foo/common` would match

Comment (geeknoid) [reply CODEOWNERS]: Done.

**Comment by geeknoid** [comment]
@howardjohn I turned codeowners off and on again, and now it's asking for code owners approval.

**Pull request merged**